src/fixtures/review.ts

```diff
@@ -20,6 +20,10 @@ import {
   MOCK_GITHUB_COMMIT_AUTHOR_TWO,
```

Comment:
What are the failure modes for this feature? Are we considering feature-flags?

You mentioned this is backward-compatible? So if we have a flag to enable this and disable the flag, the site shouldn't be affected right?

Reply:
hmm, I didn't consider feature flagging it
however, if we were to have a feature flag that is on, and then toggle it off, wouldn't there be divergence between `origin/master` and `master` if user's merge any RRs after that?

or are you suggesting to only feature flag the /compare
but not the `git pull` from `origin/master` during RR merge, so that master is always in sync?

Reply:
I thought to feature flag for a few sites then progressively expand this to 100% rollout. On growthbook,  you can whitelist for specific sites. For starters, this will be done for test sites, then 1-2 agency sites and so on

Reply:
good point! I've added back the old `compareDiff` and feature flagged it as `is_local_diff_enabled`
[9920250](https://github.com/isomerpages/isomercms-backend/pull/1172/commits/99202504538a0ac2fd9aeb7ee856f4f8bc8c4aa8)

https://app.growthbook.io/features/is_local_diff_enabled